cli/auth/api/userinfo/userinfo.go

```diff
@@ -0,0 +1,28 @@
+package userinfo
+
+import (
+	"context"
+
+	"github.com/Seiya-Tagami/pecopeco-cli/auth"
+	"github.com/Seiya-Tagami/pecopeco-cli/auth/api/model"
+	api "google.golang.org/api/oauth2/v2"
+	"google.golang.org/api/option"
+)
+
+func Get(ctx context.Context, oauth auth.OAuth) (model.Userinfo, error) {
+	client := oauth.Config.Client(ctx, oauth.Token)
```

Comment:
WithTimeoutとcancel()あったらいいかなと思いました！
```suggestion
	client := oauth.Config.Client(ctx, oauth.Token)
	ctx, cancel := context.WithTimeout(ctx, time.Second*30)
        defer cancel()
```